### ISSUE TITLE
Schematic download wrong word placement in command

### ISSUE THREAD

**Issue by Noiknez** [boy0001/FastAsyncWorldedit#1371]
**Description of the problem:**
When download a schematic with the website it place the word "schematic" in the middle and not the end, this way the command wont work, like someone say :
So for anyone with that issue, the website produces a command like:
//schematic load schematic url:d2593312-6758-42d0-ac73-b1cff20f31a6
Which should be
//schematic load url:d2593312-6758-42d0-ac73-b1cff20f31a6 schematic
instead

**How to replicate**:
Use the link https://schem.intellectualsites.com/fawe/index.php to download a schematic

**Comment by Noiknez** [comment]
With version 1.15-206 there is an error when doing the command :

```
[16:40:44]: [INFO] Naeliks issued server command: //schematic load url:0a918418-121a-48ba-b6a4-da3ab75c35ef schematic [16:40:44]: [ERROR] An unexpected error while handling a WorldEdit command 
java.lang.NullPointerException: null 
at com.sk89q.worldedit.command.SchematicCommands.load(SchematicCommands.java:215) ~[?:?] 
at com.sk89q.worldedit.command.SchematicCommandsRegistration.cmd$load(SchematicCommandsRegistration.java:267) ~[?:?] 
at org.enginehub.piston.CommandManager.execute(CommandManager.java:157) ~[?:?] 
at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommandOnCurrentThread$15(PlatformCommandManager.java:679) ~[?:?] 
at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandTask(PlatformCommandManager.java:695) ~[?:?] 
at com.sk89q.worldedit.extension.platform.PlatformCommandManager.handleCommandOnCurrentThread(PlatformCommandManager.java:681) ~[?:?] 
at com.sk89q.worldedit.extension.platform.PlatformCommandManager.lambda$handleCommand$13(PlatformCommandManager.java:651) ~[?:?] 
at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.15.2.jar:git-Paper-239] 
at org.bukkit.craftbukkit.v1_15_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:54) ~[patched_1.15.2.jar:git-Paper-239] 
at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.15.2.jar:git-Paper-239] at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?] 
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?] 
at java.lang.Thread.run(Thread.java:834) [?:?] 
```

**Comment by N0tMyFaultOG** [comment]
Your ticket has been closed because it falls into one of the following categories:

- Your issue has been fixed in a newer version of FastAsyncWorldEdit for versions 1.8 - 1.12.2, get the latest build [here](https://ci.athion.net/job/FastAsyncWorldEdit/lastSuccessfulBuild/artifact/target/) matching the platform you need.

- Your issue targets minecraft version 1.13 or higher. This is the wrong repository then. Move to [this repository](https://github.com/IntellectualSites/FastAsyncWorldEdit) and open a bug report here.
BUT BEWARE: You should consider updating to 1.15.2, using the latest Fawe build for 1.15.2 (Build 255 when writing this, get it [here](https://ci.athion.net/job/FastAsyncWorldEdit-1.15/)) and reproducing the issue in a clean environment (fresh setup Spigot or Paper server).

- Your issue has been fixed in a build of FAWE for 1.15 or an issue exists on our new repository.

- You either didn't fill out the template or deleted it.

If none of the bullet points suits, head to the second one.
